### DISCUSSION TITLE
V2 v3 변경방법

### DISCUSSION THREAD

**Discussion by rladbssam317** [changdoc/naver-vaccine-macro#221] [Q&A]
어떻게 변경하는지 알려주실수있나요?ㅠㅠ 

그리고 카카오맵에서 마커들하고 시간 기준이 업데이트가 안되어도 잘 작동되고있는것 맞죠?

**Comment by changdoc** [comment]
최신 코드에 V3로 변경 되어 있습니다.

네, 카카오 맵에서의 동작은 백그라운드로 api 호출만 하기 때문에 보여지는 마커들과 시간은 아무 상관 없이 동작합니다.
개발자 도구 network탭에 호출이 연속적으로 오류 없이 이뤄지면 정상으로 보시면 됩니다.

**Reply by rladbssam317** [reply]
![4D6CF03E-E2DB-4D1E-96A5-4EE1AD7C544F](https://user-images.githubusercontent.com/88364790/128131972-ecb72e5d-cf60-48ce-87dc-464cc8753545.jpeg)


제가 여기서 v2를 v3로 변경햇는데 이렇게 해도 되는건가요?

**Reply by changdoc** [reply]
네 최신 코드 보시면 저도 거기만 수정해둔 상태입니다.
